stable-2.4.0.md

```diff
@@ -0,0 +1,188 @@
+## stable-2.4.0
+
+This stable release introduces adds support for SMI [Traffic Split](https://github.com/deislabs/smi-spec/blob/master/traffic-split.md),
+fine-grained observability into the TLS-based identity system via the new
+`linkerd edges` command, default automatic proxy injection, installation stages,
+and a host of performance and usability improvements in the proxy and Web UI.
+
+Creating an SMI [Traffic Split](https://github.com/deislabs/smi-spec/blob/master/traffic-split.md)
+resource will allow Linkerd to split traffic between the specified backend
+services. The `linkerd edges` command provides a thorough breakdown of the name
+and identity for proxied connections, as well as an exciting new "Edges" table
+in the Web UI!
+
+Significant changes to the installation lifecycle have introduced default
+automatic proxy injection, and optional installation stages via the `linkerd
+install config` and `linkerd install control-plane` commands which separate the
+concerns for cluster-level and namespace-level privileges.
+
```

Comment:
I think the automatic proxy injection require a bit more detail, being such a big change on the UX. From the edge-19.4.5 changelog:
```
With this release the behavior of the linkerd inject command changes, where
the proxy sidecar container YAML is no longer included in its output by
default, but instead it will just add the annotations to defer the injection to
the proxy injector.
For use cases that require the full injected YAML to be output, a new
--manual flag has been added.
```

Reply:
The main difference in `inject` UX now is that previously, `linkerd inject` outputs the injected YAML blob, but now, it only outputs the annotated YAML (without the proxy YAML). The previous behaviour can be reproduced by using `linkerd inject --manual`. I am on the fence with whether to include this in the release notes. We have doc for this.